go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/osmosis-labs/osmosis/v7
+module github.com/osmosis-labs/osmosis/v8
```

Comment:
I don't think we can simply do this -- we have to update **_all_** import paths and I think @p0mvn already has a PR for this?

Reply:
That PR is closed to avoid breaking all other existing PRs. But yeah we should do this later for the entire repository

Reply:
Gotchu, will fix it